pep-0001.txt

```diff
@@ -157,21 +157,22 @@ Submitting a PEP
 ----------------
 
 Following a discussion on python-ideas, the workflow varies based on whether
-the PEP author is a core developer. If the PEP author is **not** a
-core developer then the PEP author will need to find a core developer
-*sponsor* for the PEP. The sponsor's job is to provide guidance to the PEP
-author to help them through the logistics of the PEP process (somewhat acting
-like mentor). For the core developer sponsoring, being a sponsor does **not**
-disqualify them from becoming a co-author or BDFL-Delegate later on (but not
-both). The core developer who becomes the sponsor of a PEP is recorded in the
-"Sponsor:" field of the header.
-
-Once a core developer is found that is willing to sponsor the PEP -- whether by
-being an author of the PEP or specifically a sponsor -- and deems the PEP ready
-for submission, the proposal should be submitted as a draft PEP via a
-`GitHub pull request`_.  The draft must be written in PEP style as described
-below, else it will fail review immediately (although minor errors may be
-corrected by the editors).
+the PEP author is a core developer. If the PEP author is **not** a core
+developer then the PEP author will need to find a *sponsor* for the
+PEP. Ideally, a core developer sponsor is identified, but non-core sponsors may
+also be selected with the approval of the Steering Council.  The sponsor's job
+is to provide guidance to the PEP author to help them through the logistics of
+the PEP process (somewhat acting like a mentor).  Being a sponsor does **not**
+disqualify that person from becoming a co-author or BDFL-Delegate later on (but
+not both).  The sponsor of a PEP is recorded in the "Sponsor:" field of the
+header.
+
+Once a PEP sponsor is identified -- either by being an author of the PEP, a
```

Comment:
This bit is unique in assuming that a core-dev author is a sponsor. Other parts of the text differentiate between the two (by stating that PEPs with a core-dev author don't need a sponsor.) Maybe change this to

"Once the sponsor or the core developer(s) co-authoring the PEP deem the PEP ready for submission, ..."